.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Our [Code of Conduct](https://www.classicpress.net/democracy/#democracy-conduct) and [exceptions](https://www.classicpress.net/democracy/#democracy-exceptions)
+Our code of conduct is simple: Be good to one another, take criticism well and follow the democratic process. When contributing to this repository, please note we have a [code of conduct](https://www.classicpress.net/democracy/#democracy-conduct) with some [exceptions](https://www.classicpress.net/democracy/#democracy-exceptions). 
+
+Our request is that you follow these guidelines in all your interactions with the project. 
+
+# Start helping
+You can start helping by submitting pull requests(PRs) to the develop branch. This is not limited to:
+
+## Documentation
+Good documentation will make the work of making ClassicPress a better software much faster and a better user experience.
+
+## Fixing Bugs
+Most of the bugs are shared via the [issues page](https://github.com/ClassicPress/ClassicPress/issues). When evaluating bug fixes pull requests(PRs), we look for, if not all these things
+
+  - The change impacts existing ClassicPress users. (Otherwise, there are literally thousands of things we could look at, but we need to prioritize our development time.)
+  - The change is not going to break any other use cases.
+  - The change has automated tests.
+  - We understand the change very well or can ask questions of someone who understands it very well.
+
+## New Feature Requests
+We have a number of [feature requests or petitions from our petitions website](https://petitions.classicpress.net/) submitted by different users and upvoted. It is desirable to start with the [planned](https://petitions.classicpress.net/?view=planned) or the [most wanted](https://petitions.classicpress.net/?view=most-wanted) or even submit PRs with your own suggested changes.
```

Comment:
"In case you will work on a not planned/wanted feature and doing a pull requests this doesn't means that will be approved/reviewed just because the code is available." Or something like that.
The wp experience taught me that fact, it is better to be clear form the start.

Reply:
I think there may be a nicer way to say this, but yes, I agree.

_Available code_ is not always _working code_, and even _working code_ is not always what is best for the direction of the project.